README.adoc

```diff
@@ -3,7 +3,9 @@
 
 ifdef::env-github[]
 [link=https://travis-ci.org/hawkular/hawkular-apm]
-image::https://travis-ci.org/hawkular/hawkular-apm.svg?branch=master[Build Status,70,18]
+image::https://travis-ci.org/hawkular/hawkular-apm.svg?branch=master[Build Status]
+[link=https://issues.jboss.org/browse/HWKAPM-602?filter=12328180]
```

Comment:
Jira always adds a specific issue (the first one) to that link. However, it is good suggestion link does not work in incognito mode.

This link should work:
https://issues.jboss.org/issues?jql=project%20%3D%20HWKAPM%20AND%20status%20%3D%20Open